### ISSUE TITLE
Искать потенциальные зависимости в теле функции

### ISSUE THREAD

**Issue by antoon-r** [pgcodekeeper/pgcodekeeper#25]
Сейчас не ищутся зависимости функции.

Можно разбивать тело функции на список слов и искать объекты с названием из этого списка и затем предлагать пользователю решать, какие объекты добавить в скрипт

**Comment by sgrinko** [comment]
когда-то давным давно ... был заведен такой issue :)
https://github.com/pgcodekeeper/pgcodekeeper/issues/19
тогда разработчики не захотели этим заниматься.
на данный момент времени исходный код plpgsql не парсится, но обещали в этом направлении подумать. Так что я с большим интересом жду, что скажут разработчики

**Comment by asaushkin** [comment]
Да, эта задача будет решаться после реализации парсера для plpgsql.
